src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -149,7 +175,12 @@ protected FileMonitoringController(FilePath ws) throws IOException, InterruptedE
                         // TODO is this efficient for large amounts of output? Would it be better to stream data, or return a byte[] from the callable?
```

Comment:
I agree with this comment... really really this is the place to stream it.

Reply:
See the other set of PRs which deprecate this code path anyway.